client-java/README.md

```diff
@@ -62,39 +79,37 @@ class FullTest {
+2. Uses the `ClientManager` to create a `Session` from a palisade URL. Here we are passing the user via the authority section of the URI. If a user is also passed via a property, this user in the URI takes precedence.
 3. A new Query is created by passing `QueryInfo`.
 4. The query is executed. The request is submitted to Palisade at this point and a `CompleteableFuture` is returned asynchronously. Once Palisade has processed the request, the future will emit a `Publisher` of `Messages` instances.
 5. Convert the `java.util.current.Flow.Publisher` to an RxJava `Flowable` in order to apply filtering and retrieval into a collection of `Resource` instances.
 6. Use the first resource as a test and make sure it's not null
-7. Using the session we fetch the reource. A `Download` instance is returned. At this point the request has been sent and received from the data service. The download opbject provides access to an `InputStream`. The data is not returned from the server until the input stream is first accessed.
+7. Using the session we fetch the resource. A `Download` instance is returned. At this point the request has been sent and received from the data service. The download object provides access to an `InputStream`. The data is not returned from the server until the input stream is first accessed.
 8. Using AssertJ the two input streams are checked for equality.
 
 ### Client properties
 
-__Note:__ These  properties will override any query parameters or other values within the url (e.g. port/user).
+__Note:__ These  properties will override any query parameters or other values within the URL (e.g. port/user).
 
 | Property | Description |
 | --- | --- |
 | service.user | The userId |
-| service.password | Optional passord if required |
+| service.password | Optional password if required |
 | service.palisade.port | The palisade service port. If not set will equal any port provided within the service.url |
 | service.filteredResource.port | The port for the filtered resource (websocket) service |
-| service.url | The main cluster url .e.g. pal://user@localhost:12345/cluster |
+| service.url | The main cluster URL .e.g. pal://user@localhost:12345/cluster |
 
 ### URL Query Parameters
 
-These parameters can be added to the url in the normal way.
+These parameters can be added to the URL in the normal way.
 
-__Note:__ These  parameters will be overriden by propeties 
+__Note:__ These  parameters will be overridden by properties 
 
 | Parameter | Description |
 | --- | --- |
 | port | Base port for the palisade cluster |
 | psport | The port for the Palisade Service |
-| wsport | The port for the filtered resource service. Specifiy this if it is different from the palisade service. If not supplied it will be set to that of the palisade service |
-
-
+| wsport | The port for the filtered resource service. Specify this if it is different from the palisade service. If not supplied it will be set to that of the palisade service |
```

Comment:
```suggestion
| wsport | The port for the filtered resource service. Specify this if it is different from the Palisade Service. If not supplied it will be set to that of the Palisade Service |
```

Reply:
done